Cython/Includes/libc/stdint.pxd

```diff
@@ -2,7 +2,14 @@
 # Actual compile time size used for conversions.
 
 # 7.18 Integer types <stdint.h>
-cdef extern from "<stdint.h>" nogil:
+cdef extern from * nogil:
+    """
+    #if defined(_MSC_VER) && (_MSC_VER < 1900)
+    #   include "ms_stdint.h"
```

Comment:
I think this isn't a generally available header file. My guess is that what it was originally referring to was this file:
https://github.com/pandas-dev/pandas/blob/master/pandas/_libs/src/headers/ms_stdint.h
Those are the declarations that we'd need here, or at least the ones that are listed in the Cython .pxd file below.

Reply:
Whoops. I didn't realize that it was one provided by pandas rather than Microsoft. I should have searched better. I will sift though those and try to figure out which are needed. Thank you once again for your guidance.